docs/guidelines/secret-scanning.md

```diff
@@ -0,0 +1,118 @@
+# Secret Scanning
+
+As developers, we know that secrets like passwords, API keys, and access tokens are critical to our work. But what happens when these secrets accidentally end up in our code, logs or error messages?
+
+This guideline aim to present relevant tools, some good practices for managing this risk and what to do when we have messed up. The AppSec team provides a 4 hour hands-on workshop to get started with secret scanning.
+
+- [Recommended tools](#recommended-tools)
+- [Where to scan for secrets in our SDLC](#where-to-scan-for-secrets-in-our-sdlc)
+- [What to do when we have messed up](#what-to-do-when-we-have-messed-up)
+- [How do we manage secrets in our dev environments?](#how-do-we-manage-secrets-in-our-dev-environments)
+
+## Recommended tools
+
+- Snyk Code (in IDE)
+- [Trufflehog](https://github.com/trufflesecurity/trufflehog) (elsewhere)
+
+Please note that:
+
+- False positives is a sizable challenge, scanners needs to be fine tuned to your context (detectors, testing, verification +++)
+- Incremental scans would be helpful, you may not want to scan everything all the time
+- Don't underestimate the cultural change when changing how you work. Document how your team does Secure Development, use in on-boarding, use in retrospectives - especially when you identity "interesting" pattern's.
+
+## Where to scan for secrets in our SDLC
+
+The general idea is to catch secrets in code (and other places) as early as possible. Our recommendation for most teams would be;
```

Comment:
I wouldn't use the conditional here, our recommendation either IS or IS NOT. 
Replace ; with :